org/orphaned_repos.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+comm -3 <(cat org/cloudfoundry.yml | spruce json \
+           | jq -r '.orgs.cloudfoundry.repos | with_entries(select(.value.archived != true)) | keys | map("cloudfoundry/\(.)") | sort | unique | .[]') \
```

Comment:
I prefer spruce over yq, mainly because the fast number of different version of yq out there. Also spruce is maintained by a CFF community member, which makes the risk minimal.